core/schema/module.graphqls

```diff
@@ -0,0 +1,73 @@
+scalar ModuleID
+
+extend type Query {
+  """
+  Load a module by ID, or create a new one if id is unset.
+  """
+  module(id: ModuleID): Module!
+
+  """
+  The module currently being served in the session, if any.
+  """
+  currentModule: Module
+}
+
+type Module {
+  "The ID of the module"
+  id: ModuleID!
+
+  "The name of the module"
+  name: String!
+
+  "The doc string of the module, if any"
+  description: String
+
+  "The SDK used by this module"
+  sdk: String!
```

Comment:
Should this be an Enum? With `Go`, `Node`, `Python`, (`Rust`, `Elixir`, `Dotnet`, `Java`)?